README.md

```diff
@@ -1,50 +1,49 @@
-# About
-A web application for bicycle collectives to track bicycles, bicycle work history, volunteer hours, volunteer work history, and volunteers currently in the shop.
+﻿# About
 
-See a live demo here: <http://bikeshed.wvcompletestreets.org/>  Thanks @fspc for setting that up!
+  A web application for bicycle collectives to track bicycles, bicycle work history, volunteer hours, volunteer work history, and volunteers currently in the shop.
 
-See an overview video of what the desktop view looks like and how it works here: https://www.youtube.com/watch?v=0-JjM6d9nK4.
+  See a live demo here: http://bikeshed.wvcompletestreets.org/ Thanks @fspc for setting that up!
 
-## Overview/Instructions/Guides
+  See an overview video of what the desktop view looks like and how it works here: https://www.youtube.com/watch?v=0-JjM6d9nK4.
 
-See [guides](doc/guides.md)
+  Overview/Instructions/Guides
 
-# Developer Setup
+  See guides
 
-## Running with Docker (recommended)
+#  Developer Setup
 
-1. These instructions haven't been tested, please provide corrections!
-1. Install Docker Toolbox <https://www.docker.com/toolbox>
-1. Make sure you have a machine running: `docker-machine start default && eval "$(docker-machine env default)"`
-1. Execute `docker-compose build`
-1. Execute `docker-compose run web rake db:create db:migrate db:seed`
-1. Execute `docker-compose up`
-1. If using Docker Toolbox, use `docker-machine ip default` to get the IP where the server is running.
-1. Test the Rails server is running with by visiting `<INSERT IP>:8080` in your browser.
+  Running with Docker (recommended)
 
-### Alternative Dockerfile
+  These instructions haven't been tested, please provide corrections!
+  Install Docker Toolbox https://www.docker.com/toolbox
+  Make sure you have a machine running: docker-machine start default && eval "$(docker-machine env default)"
+  Execute docker-compose build
+  Execute docker-compose run web rake db:create db:migrate db:seed
+  Execute docker-compose up
+  If using Docker Toolbox, use docker-machine ip default to get the IP where the server is running.
+  Test the Rails server is running with by visiting <INSERT IP>:8080 in your browser.
+  Alternative Dockerfile
 
-See <https://github.com/fspc/bikeshed> for an alternative docker setup.
+  See https://github.com/fspc/bikeshed for an alternative docker setup.
 
-### Developer Workflow
-The project directory should already be mounted inside the container, so you should be able to make live changes. However, since the project is running in the 'web' container, you need to prepend commands with `docker-compose run web`.
+  Developer Workflow
 
-You'll likely want to add the following aliases:
-```
```

Comment:
This is very stress. Till when you give me feedback? I am so tired.

On Wed, Jul 27, 2016 at 2:05 PM, Jason Denney <notifications@github.com>
wrote:

> In README.md
> <https://github.com/spacemunkay/BikeShed/pull/112#discussion_r72523533>:
>
> >
> > -You'll likely want to add the following aliases:
> > -```
>
> The ``` were necessary, now it looks like this which makes it difficult to
> distinguish code from text:
>
> [image: screen shot 2016-07-27 at 2 03 32 pm]
> <https://cloud.githubusercontent.com/assets/384389/17192382/14714172-5403-11e6-89a9-84b8a49d0898.png>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spacemunkay/BikeShed/pull/112/files/c9ee26ac0820b1b6749288780f296994e210f005#r72523533>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATHQ9LqVcWkpKUi3gUAFtdj_PlFjhekMks5qZ8gEgaJpZM4JTvD->
> .
>